spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/service/impl/DefaultTaskServiceTests.java

```diff
@@ -173,9 +185,10 @@ public void failOnLimitReached() {
 			initializeSuccessfulRegistry(this.appRegistry);
 			when(taskLauncher.launch(anyObject())).thenReturn("0");
 			this.launcherRepository.save(new Launcher("local", "default", taskLauncher));
-			assertEquals(10, taskService.getMaximumConcurrentTasks());
-			for (long i = 1; i <= taskService.getMaximumConcurrentTasks(); i++) {
-				assertEquals(i, taskService.executeTask(TASK_NAME_ORIG, new HashMap<>(), new LinkedList<>(), "default"));
+			assertEquals(10, taskDefinitionRetriever.getMaximumConcurrentTasks());
+			for (long i = 1; i <= taskDefinitionRetriever.getMaximumConcurrentTasks(); i++) {
+				assertEquals(i,
+						taskService.executeTask(TASK_NAME_ORIG, new HashMap<>(), new LinkedList<>(), "default"));
```

Comment:
why is this on a different line?

Reply:
because of my formatter but I re-formatted it